app/models/nomis/sentence_detail.rb

```diff
@@ -27,6 +30,20 @@ def conditional_release_date
       @conditional_release_override_date.presence || @conditional_release_date
     end
 
+    def recall_release_date
```

Comment:
Are we right to be calling this the 'recall release date'? I note that we're reading from fields '**post** recall release date'. And in the view, we present this as 'prisoner recall release date'.

I'm wondering whether both names ('post recall release date', and 'prisoner recall release date') are actually both expansions of acronym 'PRRD'..? 🤔 I'm wondering whether the name 'recall release date' is a truthful description of what this field actually represents, or if it adds to the existing mix of names.

Naming things is hard! 🙈